docs/wiki/DeploymentFlowVWAN.md

```diff
@@ -15,4 +15,4 @@ Modules in this reference implementation must be deployed in the following order
 | Order | Module                                 | Description                                                                                                                                                                                | Prerequisites                                                          | Module Documentation                                                                                                                                                  |
 | :---: | -------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 |   1   | Virtual WAN Connectivity                      | Deploys the Virtual WAN network topology and its components according to the Azure Landing Zone conceptual architecture.                                                                                          | Management Groups, Subscription for vWAN connectivity.                    | [infra-as-code/bicep/modules/vwanConnectivity](https://github.com/Azure/ALZ-Bicep/tree/main/infra-as-code/bicep/modules/vwanConnectivity)                             |
-|   2   | VNet Peering with vWAN                        | Connect a virtual network to a Virtual WAN hub.                                                                                          | Management Groups, Subscription for spoke VNet, vWAN Connectivity Module                    | _**Coming soon**_                            |
+|   2   | VNet Peering with vWAN                        | Connect a spoke virtual network to a Virtual WAN virtual hub.                                                                                          | Management Groups, Subscription for spoke VNet, vWAN Connectivity Module                    | [infra-as-code/bicep/modules/vnetPeeringVwan](https://github.com/Azure/ALZ-Bicep/tree/main/infra-as-code/bicep/modules/vnetPeeringVwan)                             |
```

Comment:
Are we missing a pre-req for a spoke VNET here also?

Reply:
Hey @faister and @jtracey93

When this PR is merged I have the https://github.com/Azure/ALZ-Bicep/issues/173 ready for review, as it will support both standard Hub & Spoke and aWAN spoke peering.

Reply:
done